packages/paste-core/components/side-modal/src/SideModal.tsx

```diff
@@ -0,0 +1,71 @@
+import * as React from 'react';
+import * as PropTypes from 'prop-types';
+import type {BoxProps} from '@twilio-paste/box';
+import {Box, safelySpreadBoxProps} from '@twilio-paste/box';
+import {styled, css} from '@twilio-paste/styling-library';
+import {StyledBase} from '@twilio-paste/theme';
+import {NonModalDialogPrimitive} from '@twilio-paste/non-modal-dialog-primitive';
+
+import {SideModalContext} from './SideModalContext';
+
+const StyledSideModal = styled.div(() =>
+  css({
+    boxShadow: 'shadow',
+    width: 'size60',
+    zIndex: 'zIndex80',
+    transform: 'none!important',
+    inset: `0rem 0rem 0rem auto!important`,
+    ['&:focus']: {
+      outline: 'none',
+    },
+    ['div:first-child']: {
+      height: '100%',
+    },
+  })
+);
```

Comment:
question: I'm being somewhat lazy, what is not possible here with `Box`? Should it be something that `Box` supports?

Reply:
Box doesn't support inset yet - this came up with minimizable dialog too but I was also being lazy and didn't want to add it.

Reply:
What's the "it"? Depending on the "it" it may or may not be worthwhile adding. They aren't super hard to add at this stage either, so if it's something that' pretty useful,  we should try and do it.

Reply:
Ok! ill add inset to box